frontend/src/components/Menu/Sidebar.vue

```diff
@@ -9,34 +9,34 @@
       <div class="mb-3 mt-3">
         <b-nav vertical class="w-200">
           <b-nav-item to="/overview" class="mb-3" active-class="activeRoute">
-            <b-icon icon="house" aria-hidden="true"></b-icon>
+            <b-img src="/img/svg/home.svg" height="20" class="svg-icon" />
```

Comment:
Can we load the svgs as icons?

Reply:
unfortunately no. bootstrap does not offer the possibility to specify a source at this point. 
but since harti wants certain icons, which we do not have. he has given me svg files ... we have to take these so